src/test/data/JsonSerializableWellNusTest/duplicateAppointmentWellNus.json

```diff
@@ -0,0 +1,57 @@
+{
+  "students" : [ {
+    "name" : "Alice Pauline",
+    "phone" : "94351253",
+    "address" : "123, Jurong West Ave 6, #08-111",
+    "tags" : [ "high" ]
+  }, {
+    "name" : "Benson Meier",
+    "phone" : "98765432",
+    "address" : "311, Clementi Ave 2, #02-25",
+    "tags" : [ "medium" ]
+  }, {
+    "name" : "Carl Kurz",
+    "phone" : "95352563",
+    "address" : "wall street",
+    "tags" : [ ]
+  }, {
+    "name" : "Daniel Meier",
+    "phone" : "87652533",
+    "address" : "10th street",
+    "tags" : [ "low" ]
+  }, {
+    "name" : "Elle Meyer",
+    "phone" : "94822240",
+    "address" : "michegan ave",
+    "tags" : [ ]
+  }, {
+    "name" : "Fiona Kunz",
+    "phone" : "94824270",
+    "address" : "little tokyo",
+    "tags" : [ ]
+  }, {
+    "name" : "George Best",
+    "phone" : "94824420",
+    "address" : "4th street",
+    "tags" : [ ]
+  } ],
+  "appointments" : [ {
+    "name" : "John Doe",
+    "date" : "2023-11-30",
+    "startTime" : "16:30",
+    "endTime" : "17:30",
+    "description" : "First Session"
+  }, {
+    "name" : "John Loh",
+    "date" : "2023-10-14",
+    "startTime" : "17:30",
+    "endTime" : "18:30",
+    "description" : "Second Session"
+  }, {
+    "name" : "John Doe",
+    "date" : "2023-11-30",
+    "startTime" : "16:30",
+    "endTime" : "17:30",
+    "description" : "First Session"
+  } ]
```

Comment:
I think it's better to match `Appointment` names with `Student` names, so this test case don't fail in future when we check Appointment must have existing student.

Reply:
okay will do it, thanks for da suggestion